Source/Shared/arcana/expected.h

```diff
@@ -69,12 +69,27 @@ namespace arcana
     class bad_expected_access : public std::exception
     {
     public:
-        const char* what() const override
+        const char* what() const noexcept override
         {
             return "tried accessing value()/error() of an expected when it wasn't set";
         }
     };
 
+    class exception_with_reason : public std::exception
```

Comment:
I'm not sure it makes sense for this one to be here. Should this be here?

Reply:
Removed from Arcana.